app/routes/session.py

```diff
@@ -301,19 +300,22 @@ def get_signed_out() -> Response | str:
 
 
 def get_runner_claims(decrypted_token: Mapping[str, Any]) -> dict:
+    runner_claims_error = "Invalid runner claims"
```

Comment:
If we can i think we should put this outside the function as a Constant near the top of the module. Also I think we should rename this variable to `INVALID_RUNNER_CLAIMS_ERROR` to be more specific